js/Game.js

```diff
@@ -11,19 +11,26 @@ class Game {
     //Start events.
     addEvents(){
         this.gameSetupButton = document.getElementById('startPageButton')
```

Comment:
Borde inte dessa HTMLElement deklareras redan i konstruktorn egentligen?

Reply:
addEvents körs från konstruktorn på sista raden "this.addEvents()" , jag lyfte ut den för ökad läsbarhet, men den kan ha ett tydligare namn typ constructorAddEvents möjligtvis.

Reply:
jag kan lyfta in den i nästa version